src/modules/inspect/inspect-correlation/repositories/inspect-correlation.repository.ts

```diff
@@ -58,6 +58,15 @@ export class InspectCorrelationRepository implements IInspectCorrelationReposito
     return logsForCorrelation;
   }
 
+  public async getLogsForProcessInstance(processModelId: string,
+                                         processInstanceId: string,
+                                         identity: IIdentity): Promise<Array<DataModels.Logging.LogEntry>> {
+    const logs: Array<DataModels.Logging.LogEntry> = await this._managementApiService
+    .getProcessInstanceLog(identity, processModelId, processInstanceId);
```

Comment:
Indentation is a bit off

```suggestion
      .getProcessInstanceLog(identity, processModelId, processInstanceId);
```